content/docs/introduction/faq.mdx

```diff
@@ -127,12 +138,12 @@ When you run `tinacms dev` locally, Tina will generate a development admin/index
 
 TinaCloud's GraphQL API returns this error when it cannot find a file in your GitHub repository. This may occur under the following circumstances:
 
-* The `tina` folder (and `__generated__` subfolder) is not in your GitHub repository remote.
-  * If the folder is in your local repository, but not in your remote, make sure there isn't a `.gitignore` file excluding it.
-* Tina is configured with a branch that doesn't exist or a branch that doesn't contain the `tina` folder.
-  * The referenced branch should be created and should contain the `tina` folder.
-* The apiURL prop is misconfigured on the TinaCMS component.
-  * Check the apiURL and make sure it looks like `https://content.tinajs.io/content/{tina_client_id}/github/{branch}` where `{tina_client_id}` matches the Client ID on the Project in TinaCloud and `{branch}` is a valid branch.
+- The `tina` folder (and `__generated__` subfolder) is not in your GitHub repository remote.
+  - If the folder is in your local repository, but not in your remote, make sure there isn't a `.gitignore` file excluding it.
+- Tina is configured with a branch that doesn't exist or a branch that doesn't contain the `tina` folder.
+  - The referenced branch should be created and should contain the `tina` folder.
+- The apiURL prop is misconfigured on the TinaCMS component.
+  - Check the apiURL and make sure it looks like `https://content.tinajs.io/content/{tina_client_id}/github/{branch}` where `{tina_client_id}` matches the Client ID on the Project in TinaCloud and `{branch}` is a valid branch.
 
 ### How do I resolve `The local GraphQL schema doesn't match the remote GraphQL schema.` errors?
```

Comment:
Hi @bradystroud 

- [ ] Change from

> How do I resolve `The local GraphQL schema doesn't match the remote GraphQL schema.` errors?

to

> How to resolve `The local GraphQL schema doesn't match the remote GraphQL schema` errors?

Reply:
Disagree - the suggestion is not a question